src/components/main/actionsPanel/nodeTreeView/NodeTreeView.tsx

```diff
@@ -221,12 +221,14 @@ const NodeTreeView = () => {
     }),
     [],
   );
-
-  const searchConfig = {
-    canSearch: false,
-    canSearchByStartingTyping: false,
-    canRename: false,
-  };
+  const searchConfig = useMemo(
```

Comment:
Why this is wrapped in a useMemo if it's going to be a static value put it outside the component. Same you can do for the dragAndDropConfig

Reply:
yeah, it will be static value, right. fixed that.